Makefile

```diff
@@ -21,6 +21,23 @@ prepare:
 	git submodule update --init --recursive
 	python gen_config.py
 
+TEAMS_DIR = static/gallery
+TEAMS = emeritus-maintainers maintainers triage-team survey-team web-team
+TEAMS_QUERY = python themes/scientific-python-hugo-theme/tools/team_query.py
+
+$(TEAMS_DIR):
+	mkdir -p $(TEAMS_DIR)
+
+$(TEAMS_DIR)/%.md: $(TEAMS_DIR)
+	$(TEAMS_QUERY) --org numpy --team "$*"  >  $(TEAMS_DIR)/$*.html
+
+teams-clean: prepare
+	for team in $(TEAMS); do \
+	  rm -f $(TEAMS_DIR)/$${team}.html ;\
+	done
+
+teams: | teams-clean $(patsubst %,$(TEAMS_DIR)/%.md,$(TEAMS))
```

Comment:
I realized recently that piped operators do not always behave the way I thought they did. Specifically, I think everything after the pipe is ordered (what I wanted) but may be evaluated as phony targets (not what I wanted). Could you please double check?

Reply:
I marked `teams` and `teams-clean` as a phony target in bb321af.  So now they both always execute regardless of how piped operators behave.